test/action_view/invalid_components_test.rb

```diff
@@ -36,6 +30,6 @@ def test_raises_error_when_more_than_one_sidecar_template_for_a_variant_is_prese
       render_inline(TooManySidecarFilesForVariantComponent)
     end
 
-    assert_includes error.message, "More than one template found for variant 'test' in TooManySidecarFilesForVariantComponent"
+    assert_includes error.message, "More than one template found for variant(s) 'test, testing' in TooManySidecarFilesForVariantComponent"
```

Comment:
What if we refactored the error message to use the `pluralize` method instead of using `(s)`?

Reply:
(and to use `to_sentence` for the list of templates)